src/main/frontend/handler.cljs

```diff
@@ -210,6 +211,7 @@
     (register-components-fns!)
     (state/set-db-restoring! true)
     (render)
+    (i18n/start)
```

Comment:
i18n has state that needs to be initialized and hence it has a start. It would be good for us to improve our app organization to consist of clearly defined components with well defined start and stop behaviors. Currently our system initialization is a mix of namespaces with blurred responsibilities and confusing names. I could make a card for this soon